src/services/ReviewService.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  EditedItemProps,
+  ReviewRequest,
+  ReviewRequestDto,
+  UserDto,
+} from "types/reviewRequest"
+
+import { apiService } from "./ApiService"
+
+export const getDiff = async (siteName: string): Promise<EditedItemProps[]> => {
+  const endpoint = `/sites/${siteName}/review/compare`
+  const { items } = await apiService
+    .get<{ items: EditedItemProps[] }>(endpoint)
+    .then(({ data }) => data)
+
+  return items
+}
+export const getCollaborators = async (
+  siteName: string
+): Promise<UserDto[]> => {
+  const endpoint = `/sites/${siteName}/collaborators`
+  return apiService
+    .get<{ collaborators: UserDto[] }>(endpoint)
+    .then(({ data }) => data.collaborators)
+}
+
+export const createReviewRequest = async (
+  siteName: string,
+  reviewData: ReviewRequestDto
+): Promise<number> => {
+  const endpoint = `/sites/${siteName}/review/request`
+  return apiService.post<number>(endpoint, reviewData).then(({ data }) => data)
+}
+
+export const getReviewRequest = async (
+  siteName: string,
+  reviewId: number
+): Promise<ReviewRequest> => {
+  const endpoint = `/sites/${siteName}/review/${reviewId}`
+  return apiService
+    .get<{ reviewRequest: ReviewRequest }>(endpoint)
+    .then(({ data }) => data.reviewRequest)
+}
+
+export const mergeReviewRequest = async (
+  siteName: string,
+  prNumber: number
+): Promise<void> => {
+  const endpoint = `/sites/${siteName}/review/${prNumber}/merge`
+  return apiService.post(endpoint)
+}
+
+export const approveReviewRequest = async (
+  siteName: string,
+  prNumber: number
+): Promise<void> => {
+  const endpoint = `/sites/${siteName}/review/${prNumber}/approve`
+  return apiService.post(endpoint)
+}
+
+export const cancelReviewRequest = async (
+  siteName: string,
+  prNumber: number
```

Comment:
I think maybe we should rename this to `reviewId` to be consistent with how it's named for the other endpoints.

Reply:
actually wdyt about changing this to be `prNumber` as we are now using `(siteName, prNumber)` to identify each RR

Reply:
Yup sounds good also, let's go ahead with `prNumber`

Reply:
could we file this as a separate issue? i think there might be downstream PRs that would also require this change, making it tedious to do here.

Reply:
Tracked in #1130.